### ISSUE TITLE
Column header using rowSelection formatter will not update after a selected row is deleted

### ISSUE THREAD

**Issue by daschws** [olifolkerd/tabulator#2574]
What i use in example: 

`table.selectRow(3);table.deleteRow(3);`

The delete seems to start on line 5993 in tabulator.js.

`Row.prototype.delete = function ()`

where on line 6025 it calls:

`_this14.deleteActual()`

Line 6037 has this block of code:

```
if (this.table.modExists('selectRow')) {
    this.table.modules.selectRow._deselectRow(this, true)
}
```

If that code gets changed to:

`this.table.modules.selectRow._deselectRow(this)`

It updates ok.

I tried this with/without passing arrays and with single/multiple ids to the table.deleteRow() method.
Both operate the same. Not sure what that change does elsewhere or if there is another way to accomplish it.
Almost all code here i used defaults from your website with no other additions.

using latest version of tabulator v4.5.3, with:

Google Chrome Version 79.0.3945.117 (Official Build) (64-bit)
FireFox v72.0.1 (64-bit)
PHP 7.4.1 (cli) (built: Dec 17 2019 19:24:02) ( ZTS Visual C++ 2017 x64 )
Window 10 v1809.build 17763.914

If you need anything, (or it works ok for you) let ne know

[https://jsfiddle.net/dsparks999/4h3wL6gq/19/](url)

[Test-Tabulator-Delete.txt](https://github.com/olifolkerd/tabulator/files/4038338/Test-Tabulator-Delete.txt)

**Comment by olifolkerd** [comment]
Hey @daschws 

Thanks for the headsup.

You are right there is definitly an issue there, but im afraid you proposed solution will break another feature.

The reason for the ***true*** being passed into that function is to prevent a redraw of the table if multi rows of the table are deleted at once (so the table is only updated once rather than multiple times.

It requires use of the blockRedraw variable that is passed into the delete function, and a small tweak to the logic in the row selection module.

I will make sure a fix is included in the next release.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @daschws 

I have pushed a fix for this to the 4.6 branch, which will be released later this weekend.

Cheers

Oli :)
